site/content/en/docs/Reference/API/Kustomization File/bases.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
I think we should have a really clear rule on how we should organize deprecated fields. Because

(1) we have so many of them already
(2) and the number of deprecated fields will increase as time goes.

Reply:
Sure! How would you like to organize them?

Would you like to open a PR on this in a Documentation Style section on the new site? For reference, said section exists
* on the current site: https://kubectl.docs.kubernetes.io/contributing/docs/
* on the new site: https://kubernetes.io/docs/contribute/style/